src/Spice86/ViewModels/CpuViewModel.cs

```diff
@@ -21,12 +22,16 @@ public partial class CpuViewModel : ViewModelBase {
     [ObservableProperty]
     private CpuFlagsInfo _flags = new();
 
-    public CpuViewModel(State state, IPauseHandler pauseHandler, IUIDispatcher uiDispatcher) {
+    [ObservableProperty]
+    private CpuStackViewModel _stack;
+
+    public CpuViewModel(State state, Stack stack, IMemory memory, IPauseHandler pauseHandler, IUIDispatcher uiDispatcher) {
         _cpuState = state;
         pauseHandler.Pausing += () => uiDispatcher.Post(() => _isPaused = true);
         _isPaused = pauseHandler.IsPaused;
         pauseHandler.Resumed += () => uiDispatcher.Post(() => _isPaused = false);
         DispatcherTimerStarter.StartNewDispatcherTimer(TimeSpan.FromMilliseconds(400), DispatcherPriority.Normal, UpdateValues);
+        Stack = new CpuStackViewModel(stack, memory, pauseHandler, uiDispatcher);
```

Comment:
I'm not sure what the reason behind adding the stack-view to the CPU window is.
Would it not be easier to add an "Open Stack View" button to the stack register? Having it in a separate window is useful to observe changes when stepping through code.

Reply:
Will do.

Reply:
CPU Stack View removed. I will do it for a later PR.